scripts/system/create/entityList/entityList.js

```diff
@@ -413,12 +413,14 @@ EntityListTool = function(shouldUseEditTabletApp) {
             alignGridToSelection();
         } else if (data.type === 'alignGridToAvatar') {
             alignGridToAvatar();
+        } else if (data.type === 'brokenUrlReport') {
```

Comment:
```suggestion
        } else if (data.type === 'brokenURLReport') {
```
Coding standard 1.1.8.